### ISSUE TITLE
GNS3 typing notes not working issue

### ISSUE THREAD

**Issue by marvaen7** [GNS3/gns3-server#2006]
i'm using GNS3 v2.2.28-1 and the same issue started from v2.2.27

**GNS3 Version : 2.2.27 & 2.2.28 
OS : Archlinux - arch kernel & Zen-kernel 
Desktop : GNOME 41** 

typing notes and editing nodes nt working at the same time i got this ouput on the console , Please check 

- exceptions.log file is very long to post it here . 

********Exception detected, traceback information saved in /home/mannaa/.config/GNS3/2.2/exceptions.log ********
```
PLEASE REPORT ON https://www.gns3.com

Traceback (most recent call last):

  File "/usr/lib/python3.10/site-packages/gns3/graphics_view.py", line 476, in mousePressEvent

    note = self.createDrawingItem("text", pos.x(), pos.y(), 2)

  File "/usr/lib/python3.10/site-packages/gns3/graphics_view.py", line 1641, in createDrawingItem

    item = TextItem(pos=QtCore.QPoint(x, y), z=z, rotation=rotation, locked=locked, project=self._topology.project(), drawing_id=drawing_id, svg=svg)

TypeError: arguments did not match any overloaded call:

  QPoint(): too many arguments

  QPoint(int, int): argument 1 has unexpected type 'float'

  QPoint(QPoint): argument 1 has unexpected type 'float'
```

**Comment by grossmj** [comment]
Fixed by https://github.com/GNS3/gns3-gui/pull/3277

Workaround before we release a new version is to use Python 3.9 or below

**Comment by marvaen7** [comment]
@grossmj , Thank you very much , problem got fixed version 2.2.29
